protocol/src/common/protocol.resolveMoniker.proposed.md

```diff
@@ -0,0 +1,28 @@
+#### Resolve Symbol Moniker (textDocument/resolveMoniker)
+
+Language Server Index Format (LSIF) introduced the concept of symbol monikers to help associate symbols across different indexes. This request adds capability for LSP server implementations to provide the same symbol moniker information given a text document position. Clients can utilize this method to get the moniker at the current location in a file user is editing and do further code navigation queries in other services that rely on LSIF indexes and link symbols together.
+
+_Client Capabilities_:
```

Comment:
Do we need any client capabilities here since there is no optional behavior per client?

Also do we need server capabilities? Is it possible to add them as part of the protocol.resolveMoniker.ts file?

Reply:
I would add a client cap with a dynamiceRegistration property to keep it consistent. 

We do need a server capabilitz to indicate whether a server will provide a moniker request handler. Otherwise the client doesn't know if it can call the server.

Reply:
Sure, will add them however I am not clear on what client capability would indicate in this case? Would it be for future use?

Reply:
It would indicate `dynamiceRegistration` for now